lib/python/treadmill_aws/cli/admin/cell/configure_data.py

```diff
@@ -70,10 +74,15 @@ def configure_data_cmd(docker_registries,
         modified = _set(data, 'realm', realm) or modified
         modified = _set(data, 'instance_profile', instance_profile) or modified
         modified = _set(data, 'subnets', subnets) or modified
+        modified = _set(data,
+                        's3_registry_region', 
+                        s3_registry_region) or modified
```

Comment:
indent.

Reply:
gah, fixed. 
Indent is correct, but EOL whitespace was removed.